File: pe-lwjgl-binding/src/main/java/org/saar/lwjgl/opengl/fbos/attachment/buffer/AttachmentBuffer.java
package org.saar.lwjgl.opengl.fbos.attachment.buffer;

import org.saar.lwjgl.opengl.fbos.attachment.AttachmentIndex;

public interface AttachmentBuffer {

    void allocate(int width, int height);

    void attachToFbo(AttachmentIndex index);

    void delete();

}

File: pe-lwjgl-binding/src/main/java/org/saar/lwjgl/opengl/fbos/attachment/buffer/AttachmentBuffer.kt
package org.saar.lwjgl.opengl.fbos.attachment.buffer

import org.saar.lwjgl.opengl.fbos.attachment.index.AttachmentIndex

interface AttachmentBuffer {
    fun allocate(width: Int, height: Int)

    fun attachToFbo(index: AttachmentIndex)

    fun delete()
}